driver/examples/plot_output.py

```diff
@@ -0,0 +1,31 @@
+import xarray as xr
+import zarr
+
+
+try:
+    import matplotlib.pyplot as plt
```

Comment:
Maybe you can put this install in `driver/examples/create_venv.sh`?

Reply:
or is the creat_venv script only supposed to run inside docker? either way, this is optional. I think most python user would know how to maplotlib anyways.

Reply:
I wanted to leave matplotlib off since it's not a "requirement" of the driver. Hopefully between the line I just added to the instructions and this error message, it should be easy enough to figure out.